srcs/requirements/Adminer/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM alpine:3.18
+
+RUN apk update && apk upgrade && apk add wget php php-session php-mysqli
+
+RUN mkdir adminer
+
+WORKDIR /adminer
+
+RUN wget http://www.adminer.org/latest.php -O index.php
+
+ENTRYPOINT [ "php","-S","0.0.0.0:8080" ]
```

```suggestion
FROM php:fpm-alpine

RUN apk update && apk upgrade && apk add wget postgresql-dev
RUN docker-php-ext-install pdo pdo_pgsql && docker-php-ext-enable pdo pdo_pgsql

RUN mkdir adminer

WORKDIR /adminer

RUN wget http://www.adminer.org/latest.php -O index.php

ENTRYPOINT [ "php","-S","0.0.0.0:8080" ]
```

malheuresement cela prend trop de temps a start (1min30 - 2min30)
le probleme de base c'est que les extensions PHP pdo_psql et psql ne sont pas installer